config/config.go

```diff
@@ -130,33 +131,34 @@ func ShipperConfigFromUnitConfig(level client.UnitLogLevel, rawConfig *proto.Uni
 	}
 
 	// We should merge config overwrites here from the -E flag,
-	// but they seem to step on the elasticsearch config,
-	// so for now, don't.
+	// but the 'path.*' variables set by elastic-agent conflict with the `path` flag in the elasticsearch settings
+	// see https://github.com/elastic/elastic-agent/issues/1729
```

Comment:
Is there a concrete example of this you can add to https://github.com/elastic/elastic-agent/issues/1729?